PygameTest/testGame.py

```diff
@@ -0,0 +1,90 @@
+# import the pygame module 
+import pygame 
+
+# import pygame.locals for easier 
+# access to key coordinates 
+from pygame.locals import *
```

Comment:
CRINGEEE `import *` is bad because then it's unclear what variables are imports and which are local